source/modules/.scaffold/scaffold.js

```diff
@@ -1,59 +1,60 @@
-/*!
+import EstaticoModule from '../../assets/js/helpers/module';
+
+const name = '{{name}}';
+
+/*
  * {{originalName}}
  *
  * @author
  * @copyright
  */
-import EstaticoModule from '../../assets/js/helpers/module';
-
-class {{className}} extends EstaticoModule {
+export default class {{className}} extends EstaticoModule {
+	static events = {
```

Comment:
(Even if not using this version, it's still better to go back to constants defined outside the class. At the moment the objects are created over and over again for every instance.)